Autobahn/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM swift:5.1
+
+RUN apt-get update && apt-get install -y libssl-dev libcurl4-openssl-dev libz-dev
+
+COPY . /WebSocketEchoServer
+
+WORKDIR /WebSocketEchoServer
+
+RUN swift build
```

Comment:
Release mode? Or not...

Reply:
Hmm good question.  I'd gone for debug mode since we're dealing with tests, but as this is essentially acceptance testing, perhaps it should be release mode.